src/NuGet.Core/NuGet.CommandLine.XPlat/Utility/MSBuildAPIUtility.cs

```diff
@@ -34,10 +34,13 @@ internal class MSBuildAPIUtility
         private const string IncludeAssets = "IncludeAssets";
         private const string PrivateAssets = "PrivateAssets";
         private const string CollectPackageReferences = "CollectPackageReferences";
+        private const string CollectCentralPackageVersions = "CollectCentralPackageVersions";
```

Comment:
We recently added a `_CollectRestoreInputs` target, so we can avoid bugs like this and no longer need to add an ever growing list of collect targets.  Let's use that instead: https://github.com/NuGet/NuGet.Client/blob/2a15947b853fcaa7804743f9e171d7824cf2f775/src/NuGet.Core/NuGet.Build.Tasks/NuGet.targets#L385-L387

Reply:
`_CollectRestoreInputs` does a lot more than what we need here, doesn't it? We just want the packageReferences with their versions